main.js

```diff
@@ -0,0 +1,36 @@
+let game = {
+    cardArray: [1, 1, 2, 2, 3, 3, 4, 4, 5, 5, 6, 6, 7, 7, 8, 8]
+}
+
+game.cardArray = shuffle(game.cardArray)
+
+// Get cards from DOM and store them as a NodeList
+var cards = document.querySelectorAll('.card')
+
+document.querySelector('.newGame').addEventListener('click', function(e) {
+    e.stopPropagation
+    let cards = document.querySelectorAll('.card')
+    cards.forEach(function(card) {
+    	card.style.backgroundImage="url('images/cardBack.jpg')"
```

Comment:
Keep whitespace consistent around the = here. Some lines have variable=value and some have variable = value. Go with the latter.

Reply:
I'd also say that using a class for hidden card (showing question mark) that gets removed when clicked would be a better solution than setting the background image to the cardBack.jpg individually. Different classes can be used to represent each card state: ie hidden, clicked, matched